stdpopsim/qc/OrySat.py

```diff
@@ -0,0 +1,105 @@
+import msprime
+import stdpopsim
+
+_species = stdpopsim.get_species("OrySat")
+
+def _BottMig_3C07_qc():
+    id = "QC-BottMig_3C07"
+    
+    #populations
+    rufipogon = stdpopsim.Population(id = "RUF")
+    indica = stdpopsim.Population(id = "IND")
+    japonica = stdpopsim.Population(id = "TRJ")
```

Comment:
These population objects need a description string, like for example:
https://github.com/popsim-consortium/stdpopsim/blob/0dcd497f8b78fcc73ede06691a53946ec6c7fd8b/stdpopsim/catalog/DroMel/demographic_models.py#L12-L18

without this, they'll error out